src/scan/path_suffix.rs

```diff
@@ -27,7 +27,7 @@ pub fn scan(command: &str) -> Vec<CommandVersion> {
 
     env::split_paths(&path)
         .filter(|p| p != &shim_dir)
-        .flat_map(|p| fs::read_dir(p).unwrap())
+        .flat_map(|p| fs::read_dir(p).ok()).flatten()
```

Comment:
Not 100% sure that this is a good idea. We'll be eating every single error.

I'm fine with failing when we can't read the dir for regular reasons: (doesn't exist, no permissions, etc.)

Should we fail when we get a weird FS error. (disk blows up or whatever)

Reply:
Looks like we can use `is_dir()`. It'll both check for the directory being present and use having the right permissions.

https://devdocs.io/rust/std/path/struct.path#method.is_dir